bark/models/behavior/motion_primitives/primitives/primitive_const_acc_change_to_left.cpp

```diff
@@ -7,7 +7,8 @@
 #include "primitive_const_acc_change_to_left.hpp"
 bark::models::behavior::primitives::PrimitiveConstAccChangeToLeft::
     PrimitiveConstAccChangeToLeft(const bark::commons::ParamsPtr& params)
-    : PrimitiveConstAccStayLane(params),
+    : BehaviorModel(params),
```

Comment:
is this needed? PrimitiveConstAccStayLane will call this constructor anyway

Reply:
yes, to support virtual inheritance for the hypothesis interface